### DISCUSSION TITLE
Offline mode

### DISCUSSION THREAD

**Discussion by conrad82** [usememos/memos#2064] [Ideas]
I use memos to quickly capture ideas, todos, ..

But in situations where I don't have internet connection it breaks down, as I cannot add or access the notes

**Is it possible to implement offline mode?** 

I recently tried https://silverbullet.md and it works offline.
Or is offline something that needs to be implemented by android app?

**Comment by boojack** [comment]
Offline mode is on our roadmap. As a workaround you can try integrating with a Telegram bot in memos, and take notes in Telegram while offline. 

Reference docs: https://usememos.com/docs/integration/telegram-bot

**Comment by dfoxg** [comment]
Any news here?
